docs: add readme template (#387)
canonical/starbase: README-template.md

Author: Is it OK to name the file this?

Reviewer: Seems fine to me. I just tested and after the first time merging in (and subsequently deleting) this template file, later merges of starbase into a project won't bring the file back again.

<If the software has approved imagery or logo, add it here.>

# Starcraft

> <Include the build status badges.>

Reviewer 2: Not sure if we care about rendering this, but if we do, the first bracketed statement after each heading doesn't show up in the output.

prospective user a sense of what the software _feels_ like.>

<This could be the constituent parts of the main user story, or smaller individual
stories. Some examples:>

- <For CLI apps and tools, three-to-five invocations of essential commands>
- <For frameworks, some short examples of the structures and syntax>
- <For libraries, the library name (`import <package>`) and the main APIs>

## Installation

<Briefly describe what platforms the software is supported on.>

<Provide the simplest way to install it, prioritising the snap release on Ubuntu.>

<If available, link to other installation methods in the documentation.>

## Documentation

<Encourage the reader to start their journey with the documentation and link to it.>

## Community and support

<Invite the user to participate in the community, with a link to the Code of Conduct and
its central forums.>

You can report any issues or bugs on the project's [GitHub
repository](https://github.com/canonical/starcraft/issues).

Starcraft is covered by the [Ubuntu Code of
Conduct](https://ubuntu.com/community/ethos/code-of-conduct).

## Contribute to Starcraft

> Starcraft is open source and part of the Canonical family. We would love your help.

Reviewer 2: ```suggestion
Starcraft is open source and part of the Canonical family. We'd love your help.
```
Style guide nitpick. Feel free to roll your eyes.

Author: This one I'm flaunting because I think separating the two words is more impactful. Similar to how we sometimes write "do not x" to emphasise how much the user shouldn't do that thing.
